policyengine_us/parameters/gov/irs/credits/estate/base.yaml

```diff
@@ -0,0 +1,12 @@
+description: The IRS provides this base exclusion amount under the estate tax credit. 
+metadata:
+  unit: currency-USD
+  label: Estate tax credit base exclusion amount  
+  period: year
+  reference:
+    - title: 26 U.S. Code § 2010 - Unified credit against estate tax - (c)(3)(A)
+      href: https://www.law.cornell.edu/uscode/text/26/2010
+values:
+  2017-01-01: 5_000_000  
```

Comment:
Lets format the files this way:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/400f534f-0f2e-4994-924e-851a00ed445d)

And lets add inflation adjustment to the file